api/providers/schema/src/models/campaign/interfaces/filters/WeekdayFilterInterface.ts

```diff
@@ -0,0 +1,3 @@
+export interface WeekdayFilterInterface {
+  weekday: number[];
```

Comment:
On pourrait utiliser un `enum` ici : `[0,1,2,3,4,5,6][]` ?

Reply:
Yes, we can